pkgs/development/python-modules/radio-beam/default.nix

```diff
@@ -21,6 +21,10 @@ buildPythonPackage rec {
     hash = "sha256-7AFkuuYLzibwwgz6zrFw0fBXCnGLzdm4OgT+Chve5jU=";
   };
 
+  # Fix distutils deprecation in Python 3.12. See:
+  # https://github.com/radio-astro-tools/radio-beam/pull/124
+  patches = [ ./fix-distutil-depr.patch ];
```

Comment:
Why not use `fetchpatch2`?

Reply:
Thanks for the suggestion. But I thought the policy was to fetch patches from the upstream repository only?  The PR on upstream isn't accepted yet.

Reply:
@dotlambda The PR has been merged upstream.